nservicebus/recoverability/index.md

```diff
@@ -0,0 +1,112 @@
+
+The Delayed Retries mechanism is implemented by rolling back the [transport transaction](/nservicebus/transports/transactions.md) and scheduling the message for [delayed-delivery](/nservicebus/messaging/delayed-delivery.md). Aborting the receive operation when transactions are turned off would result in a message loss. Therefore Delayed Retries cannot be used when transport transactions are disabled and delayed-delivery is not supported.
+
+## Fault handling
+
+When messages failed during the immediate retries and delayed retries mechanism they will be moved to the error queue. Fault handling requires a configured error queue. For more information how to configure the error queue refer to [configure error queue](/nservicebus/recoverability/configure-error-queue.md).
+
+### Transport transaction requirements
+
+Fault handling doesn't require to roll back the transport transaction. A copy of the currently handled message is sent to the configured error queue and the current transaction will be marked as successfully processed. Therefore fault handling works with all supported [transport transaction modes](/nservicebus/transports/transactions.md).
+
+## Recoverability Policy
+
+NServiceBus Version 5 and lower allowed to take full control over the delayed retries part of Recoverability only by providing a [custom delayed retries policy](/nservicebus/recoverability/configure-delayed-retries.md).
+
+Starting from Version 6 it is possible to take full control over the whole Recoverability process. For more information refer to [custom recoverability policy](/nservicebus/recoverability/custom-recoverability-policy.md). 
+
+## Total number of possible retries
+
+The total number of possible retries can be calculated with the following formula
+
+    Total Attempts = (ImmediateRetries:MaxRetries) * (DelayedRetries:NumberOfRetries + 1)
+
+So for example given a variety of FLR and SLR here are the resultant possible attempts.
+
+| ImmediateRetries:MaxRetries | DelayedRetries:NumberOfRetries | Total possible attempts |
+|-----------------------------|--------------------------------|-------------------------|
+| 1                           | 1                              | 2                       |
+| 1                           | 2                              | 3                       |
+| 1                           | 3                              | 4                       |
+| 2                           | 1                              | 4                       |
+| 3                           | 1                              | 6                       |
+| 2                           | 2                              | 6                       |
+
+NOTE: In Versions 6 and higher, the configuration of the Immediate Retries mechanism will have no effect on how many times a deferred message is dispatched when an exception is thrown. Delayed Retries will retry the message for the number of times specified in its configuration.
+
+## Retry Logging
```

Comment:
I think that sudo code with `try-catch` for retries from the first section would be better here. Or we could also move this section up.